pkg/daemon/ceph/osd/agent.go

```
@@ -159,16 +165,17 @@ func (a *OsdAgent) configureDevices(context *clusterd.Context, devices *DeviceOs
 	for _, entry := range scheme.Entries {
 		config := &osdConfig{id: entry.ID, uuid: entry.OsdUUID, configRoot: context.ConfigDir,
 			partitionScheme: entry, storeConfig: a.storeConfig, kv: a.kv, storeName: config.GetConfigStoreName(a.nodeName)}
-		err := a.startOSD(context, config)
+		osd, err := a.startOSD(context, config)
 		if err != nil {
-			return fmt.Errorf("failed to config osd %d. %+v", entry.ID, err)
+			logger.Errorf("failed to config osd %d. %+v", entry.ID, err)
```

we don't want to return the error?